bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+      description: |
+        Retrieves errors encountered during processing of unwrap transactions associated with the given hash.
+        Unwrap is only available in wrap and stake modes.
+      tags:
+        - Unwrap
+      operationId: getUnwrapErrorsByHash
+      parameters:
+        - $ref: '#/components/parameters/TransactionHash'
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Sort'
+      responses:
+        200:
+          description: Unwrap errors associated with the specified transaction hash.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UnwrapError'
+        400:
+          description: Invalid transaction hash format.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /unwrap/prepare:
+    post:
+      summary: Prepares an unwrap transaction.
+      description: |
+        Estimates the result of an unwrap operation, including fees and expected payout.
+        Unwrap operations are only available in wrap and stake modes and are not supported when the
+        bridge is configured for swap mode.
+      tags:
+        - Unwrap
+      operationId: prepareUnwrap
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PrepareRequest'
+      responses:
+        200:
+          description: Estimated unwrap transaction details, including net payout after fees.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PrepareResponse'
+        400:
+          description: Invalid unwrap preparation parameters or constraints violated.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Unexpected internal error while estimating unwrap payout.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+components:
+  parameters:
+    Address:
+      name: address
+      in: path
+      required: true
+      description: |
+        Address used for filtering wrap or unwrap requests and errors.
+        The exact format of addresses depends on the configured network (e.g. Ethereum, Symbol, NEM).
+        Clients should validate addresses according to the blockchain in use.
+      schema:
+        type: string
+        example: 0x12ab34cd56ef7890ab12cd34ef567890ab12cd34
```

Comment:
Is it possible to provide examples for each different chain?
[Using the `examples` tag instead of `example`](https://learn.openapis.org/specification/docs.html#adding-examples)?

Reply:
Added separate examples for Ethereum, Symbol and NEM.